06-dynamic-data-structures/AdvancedBracketBalance/main.c

```diff
@@ -0,0 +1,39 @@
+﻿#include <stdio.h>
+#include <locale.h>
+#include <stdlib.h>
+
+#include "../Stack/stack.h"
+#include "advancedBracketBalance.h"
+#include "userInput.h"
+#include "testsForAdvancedBracketBalance.h"
+#include "../Stack/testsForStack.h"
+
+int main(void) {
+    setlocale(LC_ALL, "Ru-ru");
+
+    bool errorCode = false;
+
+    runStackTest(&errorCode);
+    if (errorCode) {
+        return errorCode;
+    }
+
+    if (!testAdvancedBracketBalance(&errorCode) || errorCode) {
+        printf("Тесты advancedBracketBalance не пройдены\n"); {
+            errorCode = true;
+            return errorCode;
```

Comment:
```suggestion
            return -1;
```
Чему там равен errorCode после return никого волновать не будет